From: Michal Kubecek <mkubecek@suse.cz>
Date: Mon, 5 Nov 2018 13:43:02 +0100
Subject: scripts/mkmakefile: honor second argument
Patch-mainline: Never, SUSE specific
References: none

Commit 4fd61277f662 ("kbuild: do not pass $(objtree) to
scripts/mkmakefile") assumes mkmakefile is always used to generate a
makefile in current directory. This is not true when generating makefile
for our kernel-*-devel package.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 scripts/mkmakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/scripts/mkmakefile
+++ b/scripts/mkmakefile
@@ -17,7 +17,7 @@ if [ "${quiet}" != "silent_" ]; then
 	echo "  GEN     Makefile"
 fi
 
-cat << EOF > Makefile
+cat << EOF > "${2:-.}/Makefile"
 # Automatically generated by $0: don't edit
 
 ifeq ("\$(origin V)", "command line")
